src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3491,6 +3491,8 @@ export class StudyViewPageStore
         await: () => [this.selectedSamples],
         invoke: () => {
             //only invoke if there are filtered samples
+
+            console.log('this gets invoked twice. why?');
```

Comment:
comment out the the newlyAddedCharts.remove() prevents this invoke from firing twice.  the second time it fires, it reaches the ```Promise.resolve([])``` which i believe is why the data disappears from the chart.